tests/unit/test_syntax_check.py

```
@@ -0,0 +1,997 @@
+                controller_token=None,
+                controller_ssl_verify=None,
+                controller_username=None,
+                controller_password=None,
+                websocket_url=None,
+                source_dir=None,
+                filter_dir=None,
+                shutdown_delay=60,
+            )
+
+            # Should raise YAML parsing error
+            with pytest.raises(Exception):
```

Ruff (0.15.18)

[warning] Do not assert blind exception: `Exception`

(B017)